css/mobile_first.css

```diff
@@ -618,10 +622,65 @@ footer h3:hover {
   }
 
   .flex-nav-list {
-    display: none;
+    font-size: 15px;
+    display: flex;
+    position: fixed;
+    left: -150%;
+    top: -20px;
+    padding-top: 10px;
+    gap: 0;
+    background-color: var(--verde);
+    border: 3px black solid;
+    width: 100%;
+    text-align: center;
+    transition: 0.5s;
+    font-weight: 700;
+  }
+
```

Comment:
Please consider using the same set of styles for the navbar as used in the Figma template as it's a project requirement and to make your project more professional.

## Your Design:-

![image](https://user-images.githubusercontent.com/94223367/231049987-e1025a7f-c8e6-4353-b5b4-61e3c3eed712.png)


## Figma Design:-

![image](https://user-images.githubusercontent.com/94223367/231049922-7b04af0d-5323-46fd-9dcf-5f0d267db842.png)